src/api/SearchApi.js

```diff
@@ -65,6 +66,48 @@ const SEARCH_TERM :string = 'searchTerm';
 const START :string = 'start';
 
 /**
+ * `GET /search/home/{start}/{maxHits}`
+ *
+ * Executes a search over all existing entity sets to populate the home page
+ *
+ * @static
+ * @memberof loom-data.SearchApi
+ * @param {number} start
+ * @param {number} maxHits
+ * @returns {Promise}
+ *
+ * @example
+ * SearchApi.loadHomePageEntitySets(0, 10);
+ */
+
+export function loadHomePageEntitySets(start :number, maxHits :number) :Promise<> {
```

Comment:
`loom-data-js` is meant to be a general library for accessing our backend APIs, and I don't think using "Home" fits that description. "HomePage" is something specific to the application, and it doesn't belong in `loom-data-js`.

Can this be renamed to something less application specific? Is there another use case for this endpoint?